src/core/public/chrome/ui/header/home_icon.test.tsx

```diff
@@ -4,251 +4,109 @@
  */
 
 import React from 'react';
-import { mountWithIntl } from 'test_utils/enzyme_helpers';
-import { DEFAULT_MARK, DEFAULT_DARK_MARK, HomeIcon } from './home_icon';
-
-// TODO: many of these tests cover the conditional logic of which mark to load depending on raw branding configurations
-// Dark mode and custom/default fallbacks should instead be consolidated and centralized elsewhere:
-// https://github.com/opensearch-project/OpenSearch-Dashboards/issues/895#issuecomment-1164995007
```

Comment:
Is this issue closed/completed by this PR, or will it remain open until the next one?

Reply:
Open till the next one where branding is removed.